extensions/mssql/src/objectExplorerNodeProvider/fileSources.ts

```diff
@@ -94,94 +94,56 @@ export interface IHdfsFileStatus {
 }
 
 export interface IHdfsClient {
```

Comment:
Do we need this at all anymore? Can't we just directly reference WebHdfs as a class? Or put the interface in that file and have WebHDFS implement it?

Reply:
I can remove `IHdfsClient`, but wasn't there any motivation that uses interface instead of using WebHdfs directly?

Reply:
fixed.